packages/features/ee/organizations/pages/settings/other-team-members-view.tsx

```diff
@@ -136,25 +135,27 @@ const MembersView = () => {
 
   return (
     <>
-      <Meta
-        title={t("team_members")}
-        description={t("members_team_description")}
-        CTA={
-          isOrgAdminOrOwner ? (
-            <Button
-              type="button"
-              color="primary"
-              StartIcon="plus"
-              className="ml-auto"
-              onClick={() => setShowMemberInvitationModal(true)}
-              data-testid="new-member-button">
-              {t("add")}
-            </Button>
-          ) : (
-            <></>
-          )
-        }
-      />
+      {!isAppDir ? (
+        <Meta
+          title={t("team_members")}
+          description={t("members_team_description")}
+          CTA={
+            isOrgAdminOrOwner ? (
+              <Button
+                type="button"
+                color="primary"
+                StartIcon="plus"
+                className="ml-auto"
+                onClick={() => setShowMemberInvitationModal(true)}
+                data-testid="new-member-button">
+                {t("add")}
+              </Button>
+            ) : (
+              <></>
```

Comment:
Looks like we are losing the CTA for app dir?

Reply:
Oh yeah, I wrote a TODO comment; After I add it, we are good to go